File: 3.Platzi/CursoPOO/Python/main.py
from car import Car
if __name__=="__main__":
    print("Hola mundo")
    car = Car()
    car.license="AMS234"
    car.driver="Andres Herrera"
    car.passengers=4
    print(vars(car))

    car2= Car()
    car2.license="AMD123"
    car2.driver="Juan Perez"
    car2.passengers=3
    print(vars(car2))

File: 3.Platzi/CursoPOO/Java/Main.java
package Java;

class Main{
    public static void main(String[] args) {
        System.out.println("Hola mundo");
        Car car = new Car();
        car.license="AMQ123";
        car.driver="Andres Herrera";
        car.passengers=4;
        car.printDataCar();

        Car car2= new Car();
        car2.license="750BMR";
        car2.driver="Juan Perez";
        car2.passengers=3;
        car.printDataCar();
    }
}